src/main/java/org/tranqwave/fobwebapilambda/user/UpdateUserProfileLambda.java

```diff
@@ -0,0 +1,45 @@
+package org.tranqwave.fobwebapilambda.user;
+
+import static utils.ConstantUtils.ERROR;
+import static utils.ConstantUtils.SUCCESS;
+
+import com.amazonaws.services.lambda.runtime.Context;
+
+import dao.UserProfileDao;
+import dao.dbModels.DynamoDBUserProfile;
+import model.ResponseMessage;
+import model.UpdateUserProfileRequest;
+
+public class UpdateUserProfileLambda {
+    private  final UserProfileDao userProfileDao;
+
+    public  UpdateUserProfileLambda(UserProfileDao userProfileDao){
+        this.userProfileDao = userProfileDao;
+    }
+    /*Update a particular user profile details */
+
+    public ResponseMessage updateUserProfile(UpdateUserProfileRequest request, Context context){
+        final String userId = request.getEmailString();
+
+        final DynamoDBUserProfile dynamoDBUserProfile = userProfileDao.getUserProfile(userId);
+        
+        if(userProfileDao == null){
+            return new ResponseMessage(ERROR, String.format(" User profile with email %s does not exist", userId));
```

Comment:
Remove unnecessary spacing before "User" and add a period at the end of response message.